### PR TITLE
added some animations

### PR DESCRIPTION
**Pull request by avneesh0612**
I added some subtle animations on the buttons using Framer motion.

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -26,7 +27,11 @@ const Header = () => {
           </div>
         </a>
       </Link>
-      <div className="flex-none">
+      <motion.div
+        whileHover={{ scale: 1.1 }}
+        whileTap={{ scale: 0.9 }}
```
Comment (max-programming) [inline components/Header.tsx:32]: I don't think this line is needed as there is a tap animation already on the button

Comment (avneesh0612) [reply components/Header.tsx:32]: oka, I am removing that

**Comment by janaSunrise** [comment]
This looks great! Are you adding animations for the cards too?

**Comment by avneesh0612** [comment]
Hey @janaSunrise I added for the cards before but the cards are getting out of the screen :(

**Comment by janaSunrise** [comment]
Ahh, Got it. No issues 😁

**Comment by avneesh0612** [comment]
Hey @max-programming I have added the changes requested :)

**Comment by max-programming** [comment]
@avneesh0612 This is great! Thanks!

**Comment by max-programming** [comment]
@all-contributors please add @avneesh0612 for design

**Pull request merged**